ciri/README.md

```diff
@@ -0,0 +1,61 @@
+cIRI
+====
+
+A low level implementation of a IOTA node in C.
+
+This is a full-featured [IOTA](https://iota.org/) node with a convenient JSON-REST HTTP interface. It allows users to become part of the [IOTA](https://iota.org/) network as both a transaction relay and network information provider through the easy-to-use [API](https://iota.readme.io/reference).
+
+It is specially designed for users seeking a fast, efficient and fully-compatible network setup.
+
+Running an IRI node also allows light wallet users a node to directly connect to for their own wallet transactions.
```

Comment:
`allows light wallet users a node to directly connect to for` ->
`allows light wallet users to have a node with whom they can directly connect to for`